samples/attested_tls/client/enc/mbedtls_client.cpp

```diff
@@ -221,7 +212,6 @@ int launch_tls_client(char* server_name, char* server_port)
     mbedtls_pk_init(&pkey);
     oe_verifier_initialize();
 
-#ifdef ADD_TEST_CHECKING
```

Comment:
Why this is removed (how did we use this previously)?

Reply:
ADD_TEST_CHECKING is defined in the utility.h file earlier. As read and write tests are always need to be done this flag is removed.